### PR TITLE
Thumbnails fot not downloaded images

### PR DESCRIPTION
**Pull request by nasli**

### CODE REVIEW

File: Owncloud iOs Client/FileSystem/ManageThumbnails.m [hunk 1]
```diff
@@ -18,9 +18,7 @@
 #import "ManageUsersDB.h"
 #import "ManageFilesDB.h"
 #import "UIImage+Thumbnail.h"
-
-
-static NSString *thumbnailsCacheFolderName = @"thumbnails_cache";
+#import "constants.h"
```
Comment (mRoRo) [inline Owncloud iOs Client/FileSystem/ManageThumbnails.m]: constants.h import is not necessary

File: Owncloud iOs Client/FileSystem/ManageThumbnails.m [hunk 2]
```diff
@@ -100,48 +84,26 @@ - (NSString *) getThumbnailLocalSystemPathOfUserId: (NSInteger)userId {
...
 }
 
 - (void) deleteThumbnailCacheFolderOfUserId:(NSInteger) userId {
     
-    if ([[NSFileManager defaultManager] fileExistsAtPath:[self getThumbnailLocalSystemPathOfUserId: userId]]) {
+    NSString *path = [self getThumbnailLocalSystemPathOfUserId: userId];
+    if ([[NSFileManager defaultManager] fileExistsAtPath:path]) {
         NSError *error = nil;
-        [[NSFileManager defaultManager] removeItemAtPath:[self getThumbnailLocalSystemPathOfUserId: userId] error:&error];
+        [[NSFileManager defaultManager] removeItemAtPath:path error:&error];
```
Comment (mRoRo) [inline Owncloud iOs Client/FileSystem/ManageThumbnails.m]: NTH error handling

File: Owncloud iOs Client/Tabs/FileTab/DeleteFile/DeleteFile.m [hunk 3]
```diff
@@ -223,12 +223,6 @@ - (void) deleteItemFromDeviceByFileDto:(FileDto *) file {
         }
         
     } else {
-
```
Comment (mRoRo) [inline Owncloud iOs Client/Tabs/FileTab/DeleteFile/DeleteFile.m]: ManageThumbnails.h import not necessary

File: Owncloud iOs Client/Utils/FileNameUtils.m [hunk 4]
```diff
@@ -83,6 +83,26 @@ + (BOOL)isImageSupportedThisFile:(NSString*)fileName{
     
 }
 
+/*
+ * Method to know if the file is supported by the API of the thumbnail
+ * Only JPG, PNG and JPEG images files are supported for the moment.
```
Comment (mRoRo) [inline Owncloud iOs Client/Utils/FileNameUtils.m]: add GIP, BMP and MP3

File: Owncloud iOs Client/Tabs/FileTab/DeleteFile/DeleteFile.m (hunk 3 above)
Comment (nasli) [inline Owncloud iOs Client/Tabs/FileTab/DeleteFile/DeleteFile.m]: Maybe earlier it was not necessary but know it is

File: Owncloud iOs Client/Utils/InfoFileUtils.m [hunk 5]
```diff
@@ -31,6 +31,17 @@
 #import "ManageUsersDB.h"
 #import "NSObject+AssociatedObject.h"
 #import "OCSharedDto.h"
+#import "CustomCellFileAndDirectory.h"
```
Comment (mRoRo) [inline Owncloud iOs Client/Utils/InfoFileUtils.m]: CustomCellFileAndDirectory.h imported twice

**Pull request merged**